models/rnnoise_nc/README.md

```diff
@@ -0,0 +1,2 @@
+`.npy` files under this folder are tranned from project of [xiph/rnnoise](https://github.com/xiph/rnnoise), which is licensed under the BSD 3-Clause "New" or "Revised" License.  
```

Comment:
Typo? `tranned`

Reply:
BTW, about the folder naming convention, shall we use `nchw` instead? As we don't have `nc` layout in spec although rnnoise is `nc` layout, @huningxin, WDYT?